ansible_collections/arista/avd/plugins/modules/deploy_to_cv.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+
+# NOTE: This is supposed to be deprecated as per
+# https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#ansible-metadata-block
+# But our doc Jinja2 template renders it as preview which is what we want
+ANSIBLE_METADATA = {"metadata_version": "1.0", "status": ["preview"]}
+
+DOCUMENTATION = r"""
+---
+module: deploy_to_cv
+version_added: "4.6.0"
+author: Arista Ansible Team (@aristanetworks)
+short_description: Deploy various objects to CloudVision
+description: |-
+  The `arista.avd.deploy_to_cv` module is an Ansible Action Plugin providing the following capabilities:
```

Comment:
I think here it should be 
```suggestion
  The C(arista.avd.deploy_to_cv`) module is an Ansible Action Plugin providing the following capabilities:
```

to make ansible happy otherwise it does not look good in markdown

<img width="1177" alt="Screenshot 2024-01-17 at 15 24 21" src="https://github.com/aristanetworks/ansible-avd/assets/3007422/88f29eaa-c8ef-48c6-8eaa-77184ee1c089">

Reply:
fixed